internal/extproc/translator/openai_gcpanthropic.go

```diff
@@ -194,6 +184,28 @@ func translateOpenAItoAnthropicTools(openAITools []openai.Tool, openAIToolChoice
 					inputSchema.Required = requiredSlice
 				}
 
+				// Keys to skip
+				keysToSkip := map[string]bool{
```

Comment:
* Define this a global map (don't forget to add proper prefix in its variable name) to avoid allocations here
* People don't use `map[string]bool` for set in Go, but use `map[string]struct{}` instead.